### PR TITLE
dawson-three-tuts

### PR DESCRIPTION
**Pull request by GreatGhostsss**
Note proposed tags for "Installation Guide for OCI Monitoring." "Ansible" seems reasonable enough, but unsure how you may want to format "data-visualization."

### CODE REVIEW

File: ocimonitoring-w-steampipe-prometheus-grafana.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+personas:
+- backend
+date: 2021-11-03 11:42
+description: This tutorial walks you through configuring a basic OCI monitoring solution with components based on Ansible in Oracle Linux 8.
+author: 
+  name: Martin Berger
+  github: martinberger-ch
+---
+# An Installation Guide for OCI Monitoring
+**Note:** This is an experimental environment. Feel free to try it out, extend it, and have fun with it!
```
[the remark this answers is not in the document]
Comment (ttscoff) [reply ocimonitoring-w-steampipe-prometheus-grafana.md:18]: Also preferably a blank line after a headline.

**Comment by GreatGhostsss** [comment]
@ttscoff Made suggested [revisions](https://github.com/oracle-devrel/devo.tutorials/pull/34/commits/fb9065c9ac0f84f1c86881191ca1ddeba4a2f3da). I do believe this can be merged, but just wanted to run by you before doing is myself

**Comment by GreatGhostsss** [comment]
Actually, @snafuz Can this be merged? I think this advocate content and I'm still unclear if they've approved.

**Comment by GreatGhostsss** [comment]
@ttscoff Updated the shell prompts so this should be good to merge when we're able to, whenever that'll be ¯\_(ツ)_/¯

**Pull request merged**